Tools/export.sh

```diff
@@ -18,7 +18,12 @@
 #
 
 if [ -z "$SMING_HOME" ]; then
-    export SMING_HOME=$(readlink -m $BASH_SOURCE/../../Sming)
+    if [ $(basename $SHELL) = "zsh" ]; then
+        _SOURCE=${(%):-%N}
```

Comment:
@charlesschaefer Hello! I'm trying to get this running in MacOS which uses ZSH, wondering if you can shed any light? It's giving me this error:

```
sming/Tools/export.sh: line 25: ${(%):-%N}: bad substitution
```

Shell version reported as `zsh 5.9 (x86_64-apple-darwin22.0`.

Reply:
Actually, might answer my own question! If I source the script there's no error.